generator/src/main/java/com/scottlogic/deg/generator/walker/reductive/ReductiveRowSpecGenerator.java

```diff
@@ -34,7 +31,8 @@ public RowSpec createRowSpecsFromFixedValues(ReductiveState reductiveState) {
             reductiveState.getFieldValues().values().stream()
                 .collect(Collectors.toMap(
                     FieldValue::getField,
-                    fieldValue -> fieldSpecHelper.getFieldSpecForValue(fieldValue.getValue())));
+                    fieldValue -> fieldSpecHelper.getFieldSpecForValue(fieldValue.getValue())
+                                    .withFormatRestrictions(fieldValue.getFormatRestrictions(), FieldSpecSource.Empty)));
```

Comment:
You should be able to get the FieldSpecSource for the format restriction and pass it through here.

Reply:
What is FieldSpecSource? are we supporting it?